code/game/verbs/who.dm

```diff
@@ -169,4 +172,4 @@
 	if(config.show_mentors)
 		msg += "\n<b> Current Mentors ([num_mentors_online]):</b>\n" + mentmsg
 
-	src << msg
+	src << msg
```

Comment:
The whole proc needs to be checked; Bay edited it and so did we way back when TGC got Coder, I didn't realize so I copied over Bay file altogether, then back from Urist master, so I'm not sure if it'll still work properly.